packages/commonwealth/client/scripts/views/pages/discussions/thread_preview.tsx

```diff
@@ -211,10 +212,7 @@ export class ThreadPreview extends ClassComponent<ThreadPreviewAttrs> {
                   }
                 />
               </div>
-              {/* TODO Gabe 12/7/22 - Commenting out menu until we figure out fetching bug */}
```

Comment:
What was the fetching bug?

Reply:
it was the refreshing bug address in this